### ISSUE TITLE
Any way to use random color to change the color of the items once displayed on the timeline?

### ISSUE THREAD

**Issue by jjarrett21** [Lighthouse-io/react-visjs-timeline#61]
I'd like to change the color of the items in the screenshot below just random for now.

<img width="663" alt="screen shot 2019-02-22 at 10 57 14 am" src="https://user-images.githubusercontent.com/14066631/53257808-b0264780-3690-11e9-9a0a-cc85c105809b.png">

**Comment by jlsorak** [comment]
I don't think so, you could use something like https://github.com/davidmerfield/RandomColor though
